gui/static/js/components/controlpanel.js

```diff
@@ -16,20 +19,33 @@ export default class ControlPanel extends React.Component {
         this.sendMotors = this.sendMotors.bind(this);
         this.removeBot = this.removeBot.bind(this);
         this.xboxToggle = this.xboxToggle.bind(this);
+        this.getBotID = this.getBotID.bind(this);
     }
 
     handleInputChange(event) {
```

Comment:
jsdoc.

```
/**
 * Handles input change from ___.
 * @param {Event} event The event which triggered input change.
 */
handleInputChange(event) {
}
```

Reply:
this is a general handler for all input changes to modify the state